### PR TITLE
Remove deprecations alerts from sf5.4

### PR DESCRIPTION
**Pull request by maikelohcfg**

### CODE REVIEW

File: src/Uploadable/MimeTypeGuesserAdapter.php [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 class MimeTypeGuesserAdapter implements MimeTypeGuesserInterface
 {
+    /**
+     * @param $filePath
```
Comment (stof) [inline src/Uploadable/MimeTypeGuesserAdapter.php:11]: missing type

**Comment by maikelohcfg** [comment]
Hi @stof it is done

**Comment by Chris53897** [comment]
@maikelohcfg Thanks for the PR. Is there a reason the `LocaleListener` is not updated like the other 2 listeners?
Would be great if this File could be updated too.

**Comment by maikelohcfg** [comment]
> @maikelohcfg Thanks for the PR. Is there a reason the `LocaleListener` is not updated like the other 2 listeners? Would be great if this File could be updated too.

Hi Chris thanks for your revision, it is done.

**Comment by sadikoff** [comment]
Hey @stof 

It will be cool to finally merge it, looks like 7 months ago it was ready :)

**Review by stof — APPROVED** [review]

**Pull request merged**